tidb.go

```diff
@@ -243,9 +243,24 @@ func NewStore(uri string) (kv.Storage, error) {
 
 var queryStmtTable = []string{"explain", "select", "show", "execute", "describe", "desc"}
 
+func trimSQL(sql string) string {
+	// Trim space.
+	sql = strings.TrimSpace(sql)
+	// Trim leading /*comment*/
+	// There may be multiple comments
+	for strings.HasPrefix(sql, "/*") {
```

Comment:
How about strings.HasPrefix(sql, "/*") is false, then comment will not be trimed?